todo_app/app.py

```diff
@@ -1,12 +1,8 @@
-from flask import Flask
-from flask import render_template
-from flask import request
-from flask import redirect
-
-from todo_app.data.session_items import get_items
-from todo_app.data.session_items import add_item
```

Comment:
You can delete the session items file too, we will not use it again.

Reply:
Deleted.